banyan/src/forest.rs

```diff
@@ -144,9 +143,10 @@ fn zip_with_offset_ref<'a, I: IntoIterator<Item = &'a Index<T>> + 'a, T: TreeTyp
 }
 
 /// basic random access append only async tree
-impl<T> Forest<T>
+impl<T, V> Forest<T, V>
 where
     T: TreeTypes + 'static,
+    V: Serialize + DeserializeOwned + Clone + Send + Sync + Debug + 'static,
```

Comment:
why does it need to be `Sync`?

Reply:
Because if I remove it it does not compile :shrug:

Does not seem to be a problem for what I typically expect V to be, Payload or serde_cbor::Value or something else opaque.

Reply:
Then this is basically an API question: does this library hand references `&V` to other threads? It is not immediately clear to me that it should, especially since `get` returns an owned value and that `V: Clone`.